src/app/(admin)/new_dashboard/guides/page.tsx

```diff
@@ -1,26 +1,45 @@
 "use client";
 import DashboardCard from "../shared/Card";
 import PageContainer from "@/components/PageContainer";
-import { Box } from "@mui/material";
+import { Box, Typography } from "@mui/material";
 import GuideList from "./components/GuideList";
 import SearchBar from "./components/SearchBar";
-import { useSearchGuides, useGuideItems } from "./hooks";
+import { useSearchGuides, useGuides } from "./hooks";
+import { Guide } from "@/api/types/Guide";
+
 const Page = () => {
-  const { guideItems } = useGuideItems();
-  const { searchQuery, filteredGuides, handleSearchChange } =
-    useSearchGuides(guideItems);
+  const { data: guideItems, isLoading, isError, isSuccess } = useGuides();
+
+  let guides: Guide[] = [];
+
+  if (isSuccess && "data" in guideItems) {
+    guides = guideItems.data;
+  }
+
+  const { searchQuery, filteredGuides, handleSearchChange } = useSearchGuides(
+    guides
+  );
```

Comment:
if(isSuccess) just set guideItems as a success response. Why would you redefine another variable its not necessary.

Reply:
can you please wxplain about it?